lanebreach-api/app/services/sf311_case_service.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'soda/client'
+
+module Sf311CaseService
+  extend self
```

Comment:
This is neat, I didn't know about a module extending self acting as a singleton (which is the route I was going to take).